### PR TITLE
Fixed #34056 -- Updated the list of common passwords for CommonPasswordValidator.

### PR DESCRIPTION
**Pull request by pauloxnet**
Fix the issue [#34056](https://code.djangoproject.com/ticket/34056)

### CODE REVIEW

File: docs/topics/auth/passwords.txt [hunk 1]
```diff
@@ -615,12 +615,16 @@ Django includes four validators:
...
-    <https://gist.github.com/roycewilliams/281ce539915a947a23db17137d91aeb7>`_.
+    <https://gist.github.com/roycewilliams/226886fd01572964e1431ac8afc999ce>`_.
 
     The ``password_list_path`` can be set to the path of a custom file of
     common passwords. This file should contain one lowercase password per line
     and may be plain text or gzipped.
 
+    .. versionchanged:: 4.2
+
+        Updated the old deprecated list 20,000 of passwords.
```
Comment (claudep) [inline docs/topics/auth/passwords.txt:626]: Shouldn't this read `of 20,000`?

Comment (pauloxnet) [reply docs/topics/auth/passwords.txt]: Yes, thanks for the review. I've just fixed it.

**Comment by felixxm** [comment]
@pauloxnet Thanks :+1:

**Comment by pauloxnet** [comment]
@felixxm do you think it's necessary to add a note also in the release note ?

**Comment by felixxm** [comment]
> @felixxm do you think it's necessary to add a note also in the release note ?

A short note won't hurt, added.

**Pull request merged**